dev/utils/ckeditor5-dirs.js

```diff
@@ -72,18 +70,26 @@ module.exports = {
 	 * @param {String} workspacePath Absolute path to workspace.
 	 * @param {Object} packageJSON Contents of `ckeditor5` repo `package.json` file.
 	 * @param {String} ckeditor5Path Absolute path to ckeditor5 root directory.
+	 * @param {Boolean} includeRoot Include main `ckeditor5` package.
 	 * @returns {Array.<Object>}
 	 */
-	getDevDirectories( workspacePath, packageJSON, ckeditor5Path ) {
+	getDevDirectories( workspacePath, packageJSON, ckeditor5Path, includeRoot ) {
 		const directories = this.getDirectories( workspacePath );
 		const dependencies = this.getDependencies( packageJSON.dependencies );
 
+		if ( includeRoot ) {
```

Comment:
I don't like implementation of this option. The root repo doesn't have to be `ckeditor5`. What if someone forks this repo? Also, in the loop you shouldn't check `isRootDep` in every iteration – the dependencies should be correctly filled before you get to the loop (so in this condition).